nodes/ui_slider.html

```diff
@@ -112,6 +112,4 @@
     <p>The label can also be set by a message property by setting
     the field to the name of the property, for example <code>{{msg.topic}}</code>.</p>
     <p>Setting <code>msg.enabled</code> to <code>false</code> will disable the slider output.</p>
-    <p>Note: An input msg to the slider node will not change the status information displayed unless the slider
```

Comment:
Why is this here ? the slider will update if you inject a valid number with or without a following node.
please remove

Reply:
The lack of status when no nodes are connected is because no message is sent. But yes that can be fixed... fist time it has been reported.

Reply:
I am confused, the PR removes the section about the status because for me the status display is not affected by whether the output is connected.